src/_guides/language/analysis-options.md

```diff
@@ -107,37 +99,35 @@ The analyzer will use file #1 to analyze the code in `my_other_package`
 and `my_other_other_package`, and file #2 to analyze the code in
 `my_package`.
 
-## Specifying strong mode
-
-The Dart 1.x language spec supports dynamic typing, allowing you to
-write code that has no type annotations at all.
-Strong mode applies more restrictive rules to the type system and,
-as a result, finds more errors during static analysis and at runtime.
-Another benefit of strong mode is faster compilation.
-Some tools, such as dartdevc, require strong mode compliance.
 
-{% include optional-types-2.0.html %}
+## Performing additional type checks
 
-{% comment %}
-update-for-dart-2
-{% endcomment %}
-
-The simplest way to enable strong mode is to specify
-`strong-mode: true` in the analysis-options file:
+If you want stricter static checks than
+the [Dart type system][sound-dart] requires,
+consider using the `implicit-casts` and `implicit-dynamic` flags:
 
 {% prettify yaml %}
 analyzer:
-  strong-mode: true
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
 {% endprettify %}
 
-Strong mode is disabled by default. Instead of specifying `true`
-you can use the following flags to look for specific types
-of implicit casting, on top of the standard strong mode checks.
-The presence of either flag, regardless of value, enables strong mode.
+{% comment %}
+**PENDING:
+Do we still require strong-mode: true to make the analyzer using Dart 2 semantics?
+Will these flags still appear under strong-mode in Dart 2.0?
+Should we mention related command-line flags
+(--no-implicit-casts, --no-implicit-dynamic)?**
```

Comment:
For a short time, we will be living in a mixed world and I believe that strong mode will _not_ be the default. Longer term, the flag should go away and, if we still want the "implicit-casts" and "implicit-dynamic" flags they should probably be moved out from under "strong-mode".

I don't know whether those flags will remain long term, but if they do, then we should probably document them.